website/docs/d/is_share_profile.html.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: "ibm"
+page_title: "IBM : is_share_profile"
+sidebar_current: "docs-ibm-datasource-is-share-profile"
+description: |-
+  Get information about ShareProfile
+subcategory: "Virtual Private Cloud API"
+---
+
+# ibm\_is_share_profile
+
+Provides a read-only data source for ShareProfile. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_is_share_profile" "example" {
+	name = "tier-3iops"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required, string) The file share profile name.
+
+## Attribute Reference
+
+The following attributes are exported:
+
+* `id` - The unique identifier of the ShareProfile.
```

Comment:
The Id is coming like a date format in datasource output is not correct. pls check..